mypy_django_plugin/transformers/managers.py

```diff
@@ -179,9 +178,7 @@ def create_new_manager_class_from_from_queryset_method(ctx: DynamicClassDefConte
     new_manager_info.metaclass_type = new_manager_info.calculate_metaclass_type()
     # Stash the queryset fullname which was passed to .from_queryset
     # So that our 'resolve_manager_method' attribute hook can fetch the method from that QuerySet class
-    new_manager_info.metadata.setdefault("django", {})
-    new_manager_info.metadata["django"].setdefault("from_queryset_manager", {})
-    new_manager_info.metadata["django"]["from_queryset_manager"] = derived_queryset_fullname
+    new_manager_info.metadata["django"] = {"from_queryset_manager": derived_queryset_fullname}
```

Comment:
I'm guessing this slightly decreases compatibility if some other tool were to use the "django" namespace in the metadata dict?

Reply:
Yes, it does. Though the top-level metadata key is set as `"django"` throughout `django-stubs`.

Reply:
With a quick peek I saw that it's now always the case that `"django"` is included. Mypy mentions that it's intended for a plugin to have a top-level metadata key coinciding with the plugin name:

https://github.com/python/mypy/blob/254d41e10a353612d5edfdcb8b23b8e90508830b/mypy/plugin.py#L43-L48

Although, I suppose that the key should perhaps be `"django-stubs"` instead

Reply:
Right, I'm probably bike-shedding. Seems highly unlikely that there will be conflicts. And if there are, it can be addressed in the future.

Reply:
True, although to perhaps minimize custom metadata usage. It could be useful with some helper to set metadata values(?). Basically just to keep the same top-level name.

```python
def set_metadata_value(key: str, value: Any, info: TypeInfo) -> None:
    info.metadata.setdefault("django-stubs", {})
    info.metadata["django-stubs"][key] = value
```

Reply:
For reference, SQLAlchemy's mypy plugin has a utility getter/setter for `TypeInfo` metadata: https://github.com/sqlalchemy/sqlalchemy/blob/eb716884a4abcabae84a6aaba105568e925b7d27/lib/sqlalchemy/ext/mypy/util.py#L90-L95